elastic/kibana
# [Infra UI] Centralize chart components

closes [#161957](https://github.com/elastic/kibana/issues/161957)

## Summary

This PR encapsulates the styling and logic to render the Lens charts components. Doing so simplifies the usage in places that render these charts, eliminating the previously necessary code duplication


### How to test
- Start a local Kibana instance
- Navigate to `Infrastructure` > `Hosts`
- Confirm that all charts still work
  - Play with the filters
  - Verify whether only charts above the fold are rendered
  - Open the flyout
    - Switch hosts with the flyout open
